bg.js

```diff
@@ -53,23 +53,41 @@ chrome.tabs.onUpdated.addListener( ( id, changeInfo, changedTab ) =>
 			|| ( settings.listType == 'disable' && !urlIsAMatch ) )
 		{
 
-			chrome.tabs.executeScript( id, {
-				code: `document.querySelectorAll( 'link[rel*="icon"]' ).forEach( el =>
-				{
-					if ( el.oldHref )
-					{
-						return;
-					}
-					el.oldHref = el.href;
-					el.href = "${alertIcon}"
-					let showingOld = false;
-					el.interval = setInterval( () =>
-					{
-						el.href = showingOld ? "${alertIcon}" : el.oldHref;
-						showingOld = !showingOld;
-					}, ${FLASH_INTERVAL_MS} );
-				} )` 
-			} );
+			if (settings.flashIcon == 'enable' )
```

Comment:
I think to avoid code duplication it would be better to put this `if` statement right above the code that triggers the flashing, e.g.,:
```js
if (settings.flashIcon == 'enable')
{
	let showingOld = false;
	el.interval = setInterval( () =>
	{
		el.href = showingOld ? "${alertIcon}" : el.oldHref;
		showingOld = !showingOld;
	}, ${FLASH_INTERVAL_MS} );
}
```

Reply:
I agree that definitely makes sense. I struggled with that a bit initially, attempting to do the alert icon setup in two separate `executeScript()` runs (one always performed, the second conditional on `settings.flashIcon`), but that didn't really work out. 

That was before I changed the semantics of `settings.flashIcon` to having the value "enabled" or "disabled", though. With those string values, I should be able to include a conditional inside the script code, the same way `alertIcon` is used, basically:

```js
chrome.tabs.executeScript( id, {
  code: `document.querySelectorAll( 'link[rel*="icon"]' ).forEach( el =>
    {
      if ( el.oldHref )
      {
        return;
      }
      el.oldHref = el.href;
      el.href = "${alertIcon}"
      if ("${settings.flashIcon}" == 'enabled') {
        let showingOld = false;
        el.interval = setInterval( () =>
        {
          el.href = showingOld ? "${alertIcon}" : el.oldHref;
          showingOld = !showingOld;
        }, ${FLASH_INTERVAL_MS} )
      }
    } );`
```

I'll add a commit with that change (or something equivalent) later tonight.

Reply:
The most recent commit should address this.